packages/clay-date-picker/README.mdx

```diff
@@ -89,3 +97,35 @@ If you want to expand or close the picker from outside of the component, use the
 ## API
 
 <div>[APITable "clay-date-picker/src/index.tsx"]</div>
+
+### DatePicker.DateNavigation
+
+<div>[APITable "clay-date-picker/src/DateNavigation.tsx"]</div>
+
+### DatePicker.DayNumber
+
+<div>[APITable "clay-date-picker/src/DayNumber.tsx"]</div>
+
+### DatePicker.DaysTable
+
+<div>[APITable "clay-date-picker/src/DaysTable.tsx"]</div>
```

Comment:
hey @kresimir-coko, the ideal would be not to expose these APIs, because they are internal in this case we are giving the impression that we have the components `DatePicker.DateNavigation` ... available, which is not true, all the APIs available in this case are on the `index.tsx` the internal APIs, we were avoiding adding JSDoc.

I commented on this in some other issue. Many APIs that reference other interfaces are not shown because `react-docgen` cannot do this, there is a PR that has been implementing this for a long time https://github.com/reactjs/react-docgen/pull/352, I'm following but I don't think it will be merged apparently 🤷‍♂️